tests/service_test.ts

```diff
@@ -798,3 +798,50 @@ Deno.test("service - stats name respects assigned name", async () => {
 
   await cleanup(ns, nc);
 });
+
+Deno.test("service - multiple endpoints", async () => {
+  const { ns, nc } = await setup();
+  const ms = await nc.services.add({
+    name: "multi",
+    version: "0.0.1",
+    endpoint: {
```

Comment:
This should not be called "endpoint" - confusing.
I have a working name "root_subject", which I don't like, but it should be something that conveys what it is for.

Reply:
that is the current code...

Reply:
resolving this since we don't have root subject and endpoint configuration is optional, and same as `addEndoint()`.